CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [6.10.0](https://github.com/pirix-gh/ts-toolbelt/compare/v6.9.0...v6.10.0) (2020-07-03)
```

Comment:
Changelog should not be edited by hand, the version, and the changelog will update automatically thanks to the commits names you used after you run `npm run release`. This is why you should not disable hooks in the first place, you must follow the conventional commits https://www.conventionalcommits.org/en/v1.0.0/